src/chicory/backend/base.py

```
@@ -1,9 +1,14 @@
-from typing import TYPE_CHECKING, Any, Protocol
+from typing import Any, Protocol, runtime_checkable
 
-if TYPE_CHECKING:
-    from chicory.types import TaskResult, TaskState, WorkerStats
+from chicory.types import (  # noqa: TC001
```

Ruff (0.14.10)

Unused `noqa` directive (non-enabled: `TC001`)

Remove unused `noqa` directive

(RUF100)